### PR TITLE
Fix cxf logging

### PR DESCRIPTION
**Pull request by tjwatson**
- [ ] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [ ] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [ ] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

### CODE REVIEW

**Comment by tjwatson** [comment]
#build

**Comment by contbld** [comment]
Your personal pipeline request is at https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?uuid=7c514084-3208-4f6b-a2f3-cbb79d1f4da7

Target locations of links might be accessible only to IBM employees.

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.features.logging.3.2/src/org/apache/cxf/ext/logging/osgi/Activator.java [hunk 1]
```diff
@@ -121,6 +121,15 @@ public void updated(Dictionary config) throws ConfigurationException {
             }
         }
 
+        private ServiceRegistration<?> safeRegister(String name, Object service, Dictionary<String, Object> properties) {
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.features.logging.3.2/src/org/apache/cxf/ext/logging/osgi/Activator.java:127]:
```
        // Liberty Change Start
        private ServiceRegistration<?> safeRegister(String name, Object service, Dictionary<String, Object> properties) {
	
Comment

Leave a comment
            try {
                return bundleContext.registerService(name, service, properties);
            } catch (IllegalStateException e) {
                // ignore, likely the bundle is stopped
                return null;
            }
        }
        // Liberty Change End
```

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.features.logging.3.2/src/org/apache/cxf/ext/logging/osgi/Activator.java [hunk 2]
```diff
@@ -104,14 +104,14 @@ public void updated(Dictionary config) throws ConfigurationException {
             if (intentReg == null) {
                 Dictionary<String, Object> properties = new Hashtable<>();
                 properties.put("org.apache.cxf.dosgi.IntentName", "logging");
-                bundleContext.registerService(AbstractFeature.class.getName(), logging, properties);
+                intentReg = safeRegister(AbstractFeature.class.getName(), logging, properties);
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.features.logging.3.2/src/org/apache/cxf/ext/logging/osgi/Activator.java:108]:
```
// Liberty Change
intentReg = safeRegister(AbstractFeature.class.getName(), logging, properties);
```

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.features.logging.3.2/src/org/apache/cxf/ext/logging/osgi/Activator.java [hunk 3]
```diff
@@ -104,14 +104,14 @@ public void updated(Dictionary config) throws ConfigurationException {
...
             }
 
             if (enabled) {
                 if (serviceReg == null) {
                     Dictionary<String, Object> properties = new Hashtable<>();
                     properties.put("name", "logging");
-                    serviceReg = bundleContext.registerService(Feature.class.getName(), logging, properties);
+                    serviceReg = safeRegister(Feature.class.getName(), logging, properties);
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.features.logging.3.2/src/org/apache/cxf/ext/logging/osgi/Activator.java:116]: // Liberty Change
intentReg = safeRegister(AbstractFeature.class.getName(), logging, properties);

**Comment by contbld** [comment]
<h3>Code analysis and actions</h3>
DO NOT DELETE THIS COMMENT.

- 1 product code files were changed.

**Review by neuwerk — CHANGES_REQUESTED** [review]

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.features.logging.3.2/src/org/apache/cxf/ext/logging/osgi/Activator.java (hunk 2 above)
Comment (tjwatson) [reply dev/com.ibm.ws.org.apache.cxf.cxf.rt.features.logging.3.2/src/org/apache/cxf/ext/logging/osgi/Activator.java:108]: Done

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.features.logging.3.2/src/org/apache/cxf/ext/logging/osgi/Activator.java (hunk 3 above)
Comment (tjwatson) [reply dev/com.ibm.ws.org.apache.cxf.cxf.rt.features.logging.3.2/src/org/apache/cxf/ext/logging/osgi/Activator.java:116]: Done

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.features.logging.3.2/src/org/apache/cxf/ext/logging/osgi/Activator.java (hunk 1 above)
Comment (tjwatson) [reply dev/com.ibm.ws.org.apache.cxf.cxf.rt.features.logging.3.2/src/org/apache/cxf/ext/logging/osgi/Activator.java:127]: Done

**Comment by tjwatson** [comment]
#libby

**Comment by contbld** [comment]
Your personal pipeline request is at https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?uuid=2d3f374f-4d0f-4b28-bfa0-6bc9ba87c2bb

Target locations of links might be accessible only to IBM employees.

**Review by neuwerk — APPROVED** [review]

**Comment by contbld** [comment]
<h3>Code analysis and actions</h3>
DO NOT DELETE THIS COMMENT.

- 1 product code files were changed.

**Comment by contbld** [comment]
https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?uuid=2d3f374f-4d0f-4b28-bfa0-6bc9ba87c2bb completed successfully!

**Comment by contbld** [comment]
https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?uuid=7c514084-3208-4f6b-a2f3-cbb79d1f4da7 completed successfully!

**Pull request merged**